### PR TITLE
fix(SPDX):  Import SPDX licenses with new SPDX library (tools-java 1.0.4)

### PR DESCRIPTION
**Pull request by tuannn2**
[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.
> * Which issue is this pull request belonging to and how is it solving it? (*Refer to issue here*)
> * Did you add or update any new dependencies that are required for your change?

Issue: #1699

### How To Test?
1. Go to License Page in Tab Admin
2. Click button Import SPDX Information
3. Check License in tab License in Home Page

### Checklist
Must:
- [ ] All related issues are referenced in commit messages and in PR

### CODE REVIEW

**Comment by KoukiHama** [comment]
Is this PR  fix "obligation import" and "license import" ?

**Comment by KoukiHama** [comment]
I am not sure why the test Import File?
#1699 mention about  "Import SPDX licenses" button in admin/license page, not importing 
spdx ref file.
@tuannn2

**Comment by tienlee** [comment]
> I am not sure why the test Import File? #1699 mention about "Import SPDX licenses" button in admin/license page, not importing spdx rdf file. @tuannn2

Issue 1699 was found by @shi9qiu . I will discuss with @tuannn2 and @shi9qiu , then feedback you tomorrow.

**Comment by tuannn2** [comment]
Thank you @KoukiHama,
I'm a bit confused. Really don't need to import the File, just test and click Import SPDX Information.

File: backend/src/src-licenses/pom.xml [hunk 1]
```diff
@@ -31,7 +31,14 @@
...
         <dependency>
             <groupId>org.spdx</groupId>
-            <artifactId>spdx-tools</artifactId>
+            <artifactId>tools-java</artifactId>
+        </dependency>
+        <!-- https://mvnrepository.com/artifact/com.googlecode.json-simple/json-simple -->
+        <dependency>
+            <groupId>com.googlecode.json-simple</groupId>
+            <artifactId>json-simple</artifactId>
+            <version>1.1.1</version>
```
Comment (akapti) [inline backend/src/src-licenses/pom.xml:40]: Kindly define the version in parent (main) `pom.xml` file, and include it here.
also consider removing the maven repo link in comment on line `36`.

File: backend/src/src-licenseinfo/pom.xml [hunk 2]
```diff
@@ -29,23 +29,15 @@
...
-                    <artifactId>slf4j-log4j12</artifactId>
-                </exclusion>
-            </exclusions>
+            <artifactId>tools-java</artifactId>
+        </dependency>
+        <!-- https://mvnrepository.com/artifact/com.googlecode.json-simple/json-simple -->
+        <dependency>
+            <groupId>com.googlecode.json-simple</groupId>
+            <artifactId>json-simple</artifactId>
+            <version>1.1.1</version>
```
Comment (akapti) [inline backend/src/src-licenseinfo/pom.xml:38]: Kindly define the version in parent (main) `pom.xml` file, and include it here.
also consider removing the maven repo link in comment on line `34`.

**Review by akapti — CHANGES_REQUESTED** [review]
Tested the changes in local, and working fine.
suggested a review comment, please incorporate.

screenshot for reference:
image

**Review by akapti — APPROVED** [review]
Looks good to me.
tested and successfully imported SPDX licenses.
image

**Pull request merged**